### ISSUE TITLE
Does not return in time when invoked in batch file.

### ISSUE THREAD

**Issue by SainoNamkho** [microsoft/terminal#4709]
# Environment

```none
Windows build number:  10.0.18362.0
Windows Terminal version: 0.9.433.0
```

# Steps to reproduce

Create a batch file `a.bat` with content
```cmd
wt & echo OK!
```
Then open the batch file.

# Expected behavior

As invoked in conhost or terminal, the first console types "OK!" immediately when the new `wt` window opens.
![t1](https://user-images.githubusercontent.com/23036788/75147741-fe08dc00-5738-11ea-847b-71b1d10693db.JPG)

# Actual behavior

"OK!" is not printed until the new `wt` windows is closed.
![image](https://user-images.githubusercontent.com/23036788/75147932-6952ae00-5739-11ea-8bff-b34fc5009a4e.png)
![image](https://user-images.githubusercontent.com/23036788/75147971-7cfe1480-5739-11ea-90ad-4f7ec3aaf753.png)

**Comment by eryksun** [comment]
On the command line, cmd doesn't wait for GUI programs. But in batch mode it always waits. Use `start` to run without waiting.

**Comment by SainoNamkho** [comment]
> On the command line, cmd doesn't wait for GUI programs. But in batch mode it always waits. Use `start` to run without waiting.

But it doesn't wait for other GUI programs (I've tested explorer.exe, vscode and chrome), is there any thing special with this terminal?

**Comment by eryksun** [comment]
> But it doesn't wait for other GUI programs (I've tested explorer.exe, vscode and chrome), is there any thing special with this terminal?

You're testing programs that use a launcher to spawn another process. The launcher exits immediately, and the batch script continues. Try something simpler, like notepad.

**Comment by DHowett-MSFT** [comment]
It's weird how these things come in cycles. This is actually a /dup of #4702, which was filed only 7 issues ago.

**Comment by DHowett-MSFT** [comment]
Actually, good call on @eryksun's part: notepad causes the same behavior. Closed #4702 as by-design. :smile:
